libatomic_ops/plan.sh

```diff
@@ -0,0 +1,18 @@
+pkg_name=libatomic_ops
```

Comment:
This made me think of https://www.youtube.com/watch?v=O_WLw_0DFQQ so I had to put the link here.